wp-content/themes/reconasia/assets/_scss/layout/_footer.scss

```diff
@@ -60,31 +72,36 @@
   &__logo {
     grid-area: logo;
     max-width: 300px;
+  }
 
-    &,
-    &:visited {
-      // color: $color-text-gray-10;
-    }
+  &__desc {
+    grid-area: csisdesc;
   }
 
   .widget:first-of-type {
-    grid-area: csisdesc;
+    margin-bottom: rem(24);
   }
```

Comment:
Let's move this into the `&__desc` selector, so this only effects the widget within the description.